### PR TITLE
Add audit and risk QA

### PR DESCRIPTION
**Pull request by tomquirk**
## What does this PR do and why?

Closes https://github.com/jbx-protocol/juice-interface/issues/1391

- removes `unaudited` for any v2 UI
- Adds "has jb been audited" section to the QA

## Screenshots or screen recordings

Screen Shot 2022-07-30 at 7 30 48 AM

## Acceptance checklist

- [x] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [x] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [x] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

**Comment by tomquirk** [comment]
@peripheralist could you pls take a look at this? 🙏

File: src/pages/home/QAs.tsx [hunk 1]
```diff
@@ -91,7 +92,7 @@ export default function QAs(): {
         <Trans>
           Each project has its own{' '}
           <ExternalLink href="https://youtu.be/cqZhNzZoMh8">
-            ERC-20 tokens
```
Comment (peripheralist) [inline src/pages/home/QAs.tsx:94]: nice 👌

File: src/pages/home/QAs.tsx [hunk 2]
```diff
@@ -315,6 +322,29 @@ export default function QAs(): {
         </Trans>
       ),
     },
+    {
+      q: <Trans>Has Juicebox been audited?</Trans>,
```
Comment (peripheralist) [inline src/pages/home/QAs.tsx:326]: awesome 🙏

File: src/pages/home/HowItWorksSection.tsx [hunk 3]
```diff
@@ -110,8 +110,8 @@ export function HowItWorksSection() {
               <p>
                 <InfoCircleOutlined />{' '}
                 <Trans>
-                  Juicebox is new, unaudited, and not guaranteed to work
-                  perfectly. Before spending money, do your own research:{' '}
+                  Juicebox is new and not guaranteed to work perfectly. Before
```
Comment (peripheralist) [inline src/pages/home/HowItWorksSection.tsx:113]: can we also remove "new"? maybe something like

> Juicebox is not guaranteed to be free of bugs or exploits. Before spending money, do your own research:

Comment (tomquirk) [reply src/pages/home/HowItWorksSection.tsx]: Yep great call. Looks like dis now.

Screen Shot 2022-07-30 at 10 47 48 AM

**Comment by tomquirk** [comment]
wdyt @peripheralist ? 🙏

**Review by peripheralist — APPROVED** [review]
yessirrrr 🙏

**Pull request merged**